web/server.js

```diff
@@ -39,31 +38,42 @@ module.exports = function(options) {
 
   var server = new Hapi.Server(serverConfig);
 
-  server.connection({
-    host: options.host,
-    port: options.port
-  });
-
   if ( options.logging ) {
-    server.register({
-      register: require('hapi-bunyan'),
+    await server.register({
```

Comment:
changed from `hapibunyan` to `good-bunyan`. The logging looks kinda gross but it works.

Reply:
Tip: `npm install -g bunyan` then `npm start | bunyan` will format the output a bit better for debugging.